### DISCUSSION TITLE
Is it possible to install EU as well as RoW maps

### DISCUSSION THREAD

**Discussion by mrsdbskn** [Mr-MIBonk/M.I.B._More-Incredible-Bash#447] [Q&A]
Hi all, I have an MIB2 High unit with the Europe region map installed. I'm planning to drive to Turkey, which is outside the coverage area of my current map. Can I add the Turkey map data to my existing Europe map or is there another way to use Audi navigation in Turkey?

**Comment by jonnypuma** [comment]
Wondering the same if you can install several packs on top of each other.
